worlds/cv64/options.py

```diff
@@ -508,7 +508,7 @@ class WindowColorA(Range):
 
 class DeathLink(Choice):
     """
-    When you die, everyone dies. Of course the reverse is true too.
+    When you die, everyone who enabled death link dies. Of course, the reverse is true too.
```

Comment:
Although only HK, CV64, and OC2 call it "deathlink" (no space), they are all internally consistent:
```suggestion
    When you die, everyone who enabled deathlink dies. Of course, the reverse is true too.
```

Reply:
At first i commited that, but then i read the actual yaml and saw that the option itself is modern `death_link`, so instead i edited all mentions of `DeathLink` with `death link`. If you are fine with that, you can resolve this conversation

Reply:
Then you need to change the display name too

Reply:
is that applied in spoiler log?

Reply:
ah, it's both on the website and in the spoiler. Changed